test/envvar/build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    let value = env!("REQUIRED_VARIABLE");
+    if value != "EXPECTED_VALUE" {
+        panic!("unexpected value for required environment variable");
+    }
+    if env!("ANOTHER_VARIABLE") != "ANOTHER_VALUE" {
```

Comment:
maybe check with std::env::var.

I suggest this:

```rust
assert_eq!(env!("REQUIRED_VARIABLE"), "EXPECTED_VALUE");
assert_eq!(std::env::var("ANOTHER_VARIABLE").unwrap(),  "ANOTHER_VALUE");
```

Reply:
Sure, done